feature-libs/user/profile/core/facade/facade-providers.ts

```diff
@@ -0,0 +1,33 @@
+import { UserEmailService } from './user-email.service';
+import { UserPasswordService } from './user-password.service';
+import { UserProfileService } from './user-profile.service';
+import { UserRegisterService } from './user-register.service';
+import {
+  UserEmailFacade,
+  UserPasswordFacade,
+  UserProfileFacade,
+  UserRegisterFacade,
+} from '@spartacus/user/profile/root';
+
+export const facadeProviders = [
```

Comment:
Why not type this array to `Provider[]`?
Same goes for other providers.ts files, e.g. feature-libs/user/account/core/facade/facade-providers.ts

Reply:
ok